ComputerVision/SDK/AnalyzeImage/Program.cs

```diff
@@ -107,10 +108,9 @@ private static void DisplayFaceResults(ImageAnalysis analysis)
         private static void DisplayAdultResults(ImageAnalysis analysis)
         {
             //racy content
-            Console.WriteLine("Adult:");
-            Console.WriteLine("Adult score = {0} ({1}), racy score = {2} ({3})",
-                analysis.Adult.AdultScore, analysis.Adult.IsAdultContent,
-                analysis.Adult.RacyScore, analysis.Adult.IsRacyContent);
+            Console.WriteLine("Adult:"); 
+            Console.WriteLine("Has adult content? : {0} Confidence : {1}", analysis.Adult.IsAdultContent, analysis.Adult.AdultScore);
```

Comment:
Please remove extra spaces before ':' on each of these results. I would also suggest making the headers more or a statement than a question (e.g. "Is adult content: {0}, Confidence: {1}").